apps/server/swagger/data-api.yml

```diff
@@ -148,3 +148,47 @@
         $ref: '#/components/responses/ServerError'
       503:
         $ref: '#/components/responses/ServiceUnavailableError'
+
+/data/{systemId}:
+  delete:
+    summary: Delete Submitted Data by System ID
+    tags:
+      - admin
+    parameters:
+      - name: systemId
+        in: path
+        required: true
+        schema:
+          type: string
+      - name: dryRun
+        description: If true won't execute the deletion, but will return what will be deleted, default is false.
+        in: query
+        required: false
+        schema:
+          default: false
+          type: boolean
+      - name: reason
+        description: Reason why the record is deleted
+        in: query
+        required: false
```

Comment:
should this be required? or is there a default reason given if one is not provided?

Reply:
There is no default `reason`. In a happy scenario the user should be the responsible to define the reason for a deletion, however will leave this query param as **required** as a safe check

Reply:
renamed to `comment` this value is stored as a `comment` on an Audit table